draft-ietf-ppm-dap.md

```diff
@@ -1152,48 +1152,26 @@ The `PrepareStep` is structured as follows:
 
 ~~~
 enum {
-  initialize(0),
-  continue(1),
-  finished(2)
-  reject(3),
+  continue(0),
+  finished(1)
+  reject(2),
   (255)
 } PrepareStepState;
 
 struct {
+  ReportId report_id;
   PrepareStepState prepare_step_state;
   select (PrepareStep.prepare_step_state) {
-    case initialize:
-      ReportShare report_share;
-      opaque payload<0..2^32-1>;
-    case continue:
-      ReportId report_id;
-      opaque payload<0..2^32-1>;
-    case finished:
-      ReportId report_id;
-    case reject:
-      ReportId report_id;
-      ReportShareError report_share_error;
+    case continue: opaque payload<0..2^32-1>;
+    case reject:   ReportShareError report_share_error;
```

Comment:
RFC 8446 presentation language omits cases which have no message, see e.g. https://datatracker.ietf.org/doc/html/rfc8446#appendix-B.3.

Reply:
I made things consistent in the other direction, by removing "empty" cases.

Reply:
SGTM. Can you please add a note to the commit message this, as well as a reference to the text in RFC 8446? The reason is that @wangshan previously added the "Empty" bit, and since we're reverting that change we need to at least document why it's valid.